src/main/java/Task.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Represents a task.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    char type;
+    String unparseMessage;
```

Comment:
The naming of this variable might make it seem like it is an action and can be called.